src/EnrollU2FController.js

```diff
@@ -63,7 +65,13 @@ function (Okta, FormType, FormController, Footer, Q) {
           u2f.register(appId, registerRequests, [], function (data) {
             self.trigger('errors:clear');
             if (data.errorCode && data.errorCode !== 0) {
-              deferred.reject({ responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}});
+              if (data.errorCode === 5) { // TIMEOUT
+                self.trigger('abort');
+                deferred.resolve(transaction);
+              }
+              else {
+                deferred.reject({xhr: {responseJSON: {errorSummary: 'Error Code: ' + data.errorCode}}});
```

Comment:
This case, I need texts to localize them.

1 - OTHER_ERROR 
2 - BAD_REQUEST 
3 - CONFIGURATION_UNSUPPORTED
4 - DEVICE_INELIGIBLE 
5 - TIMEOUT 

Codes 1,2,3 depend on the configuration and can be localized with one common message.
For the Code 4 I have a key in the i18n, but don't have texts from @vincentvoong-okta 
For the Code 5 we can avoid showing the error and just reset the form.